src/processes/follow-topic-process.js

```diff
@@ -0,0 +1,72 @@
+const StreamChat = require('stream-chat').StreamChat;
+
+const {LogError, UserTopic, CommunityMessageFormat} = require('../databases/models');
+
+const followTopicProcess = async (job, done) => {
+  try {
+    console.info('running job register process ! with id ' + job.id);
+    const serverClient = StreamChat.getInstance(process.env.API_KEY, process.env.SECRET);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Sensitive data like API keys and secrets should not be accessed directly via `process.env`. Instead, consider using a configuration management library like `dotenv` or `config` to manage environment variables.

```diff
- const serverClient = StreamChat.getInstance(process.env.API_KEY, process.env.SECRET);
+ const serverClient = StreamChat.getInstance(config.get('API_KEY'), config.get('SECRET'));
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->